test/byContract/IexecPoco/IexecPocoDelegate.test.ts

```diff
@@ -236,6 +236,16 @@ describe('IexecPocoDelegate', function () {
             // Verify stored deal
             expect((await viewDeal(dealId)).sponsor).to.equal(sponsor.address);
         });
+
+        it('Should get empty task', async function () {
+            // Covers `viewTask` in tests.
+            // Fixes Codecov issue with IexecPocoAccessorsDelegate.
+            const task = await IexecPocoAccessorsDelegate__factory.connect(
+                iexecPocoInstance.address,
+                anyone,
+            ).viewTask(ethers.utils.randomBytes(32));
+            expect(task.dealid).to.equal(ethers.constants.HashZero);
```

Comment:
Something like:
```suggestion
            const { orders } = buildOrders({
                assets: ordersAssets,
                requester: requester.address
            });
            const { dealId, taskId, taskIndex, startTime } =
                await iexecWrapper.signAndMatchOrders(orders);
            await iexecPocoAsAnyone.initialize(dealId, taskIndex).then(tx => tx.wait())
            
            const task = await IexecPocoAccessorsDelegate__factory.connect(
                iexecPocoInstance.address,
                anyone,
            ).viewTask(taskId);
            expect(task.dealid).to.equal(dealId);
```
?

Reply:
Check this [Add get task test](https://github.com/iExecBlockchainComputing/PoCo/pull/81/commits/7bc72774f2667244bcd8084d7d8d025ee197336d)